### ISSUE TITLE
Modal USWDS Pass Through to NCIDS

### ISSUE THREAD

**Issue by andyvanavery** [NCIOCPL/ncids#864]
### USWDS Component Name
[Modal](https://designsystem.digital.gov/components/modal/)

### USWDS Component Packages
* Package usage: `@forward "usa-modal";`
* Dependencies: `uswds-fonts`, `usa-button`

### Subtasks
- [ ] Make a folder with the component name under uswds-native
- [ ] Copy USWDS twig and any JSON
- [ ] Make an index.scss for the stories
- [ ] Copy and hack up USWDS stories.js file to use our story format
- [ ] Create scenarios.js by adding a scenario for each story in stories.js
- [ ] Add link to the USWDS documentation on the NCIDS documentation site

**Comment by olitharp-nci** [comment]
blocked by #845 and #847 (a story requires these components)

**Comment by bennettcc** [comment]
both dependencies have been completed (#845 and #847) - moving out of on hold
